docs/source/README.rst

```diff
@@ -0,0 +1,39 @@
+.. _README:
```

Comment:
a symlink might not work well on windows. what ever ends up with out duplicate files would be great.